world-chain-builder/crates/world/node/tests/e2e.rs

```diff
@@ -312,7 +312,7 @@ async fn test_dup_pbh_nonce() -> eyre::Result<()> {
 
     let raw_tx_0 = ctx.raw_pbh_tx_bytes(signer.clone(), 0, 0).await;
     ctx.node.rpc.inject_tx(raw_tx_0.clone()).await?;
-    let raw_tx_1 = ctx.raw_pbh_tx_bytes(signer.clone(), 0, 1).await;
+    let raw_tx_1 = ctx.raw_pbh_tx_bytes(signer.clone(), 0, 0).await;
```

Comment:
```suggestion
    let raw_tx_1 = ctx.raw_pbh_tx_bytes(signer.clone(), 0, 1).await;
```

This is the nonce, and should be 1 I believe as it's on the same signer

Reply:
Yes, we are testing that submitting a duplicate nonce fails. This is set to 0 so that the following line clears.

`assert!(ctx.node.rpc.inject_tx(raw_tx_1.clone()).await.is_err())`

Reply:
Ahh my bad, sounds good!